prebid-mobile/privacy-regulation.md

```diff
@@ -36,12 +36,12 @@ sidebarType: 2
 
 ### Framework APIs
 
-Prebid Mobile provides two APIs for app publishers to use with the Framework. These APIs allow you to:
+Prebid Mobile provides three APIs for app publishers to use with the Framework. These APIs allow you to:
 
--   Define whether European privacy regulations should apply
+-   Define whether the user is located in the European Economic Area (the "EEA") and if European privacy regulations should apply
 -   Set the [IAB Europe](https://www.iabeurope.eu/) (IAB) consent string
 
-This information will be persisted by Prebid Mobile and will be added to each ad call. Publishers/Consent Management Platforms (CMPs) are free to store these values in an `NSUserDefaults/SharedPreferences` interface (as defined by [Mobile In-App CMP API v1.0: Transparency & Consent Framework](https://github.com/InteractiveAdvertisingBureau/GDPR-Transparency-and-Consent-Framework/blob/master/Mobile%20In-App%20Consent%20APIs%20v1.0%20Final.md)) instead of passing them via the new APIs, and Prebid Mobile will read the values as a fallback.
+This information will be persisted by Prebid Mobile and will be added to each ad call to the demand partners. Publishers/Consent Management Platforms (CMPs) are free to store these values in an `NSUserDefaults/SharedPreferences` interface (as defined by [Mobile In-App CMP API v1.0: Transparency & Consent Framework](https://github.com/InteractiveAdvertisingBureau/GDPR-Transparency-and-Consent-Framework/blob/master/Mobile%20In-App%20Consent%20APIs%20v1.0%20Final.md)) instead of passing them via the new APIs, and Prebid Mobile will read the values as a fallback.
```

Comment:
We are missing references to the TCF2.0 framework per the draft document we have under review: https://docs.google.com/document/d/1glzKEDlrRPq1JBKX33vyRjk_Unx2rfu2ZYne8piLkp8/edit#